doc/source/reference/c-api.types-and-structures.rst

```diff
@@ -871,8 +871,8 @@ PyArrayIter_Type
    .. c:member:: npy_intp *PyArrayIterObject.backstrides
 
        How many bytes needed to jump from the end of a dimension back
-       to its beginning. Note that *backstrides* [k]= *strides* [k]*d
-       *ims_m1* [k], but it is stored here as an optimization.
+       to its beginning. Note that `backstrides`[k]= `strides`[k] *
+       `dims_m1`[k], but it is stored here as an optimization.
```

Comment:
This doesn't render correctly. You can put single backticks around the whole expression, possibly with a `:code:` preceding, e.g.
```
       How many bytes needed to jump from the end of a dimension back
       to its beginning. Note that :code:`backstrides[k]= strides[k] *
       dims_m1[k]`, but it is stored here as an optimization.
```
You can check things by changing to the `doc` directory, doing `make html` and then viewing `build/html/index.html` in a browser.